npm_and_yarn/Dockerfile

```diff
@@ -16,6 +16,10 @@ RUN curl -sL https://deb.nodesource.com/setup_16.x | bash - \
 RUN corepack enable \
   && corepack prepare yarn@$YARN_VERSION --activate
 
+# Install pnpm and set it to a stable version
+RUN corepack enable \
+  && corepack prepare pnpm@8.2.0 --activate
```

Comment:
I know this PR is in the early stage, but just to mention, `pnpm` major versions are most often not backward compatible.

If you have a project running pnpm v7 and tries to install deps with v8, it won't work. They use the `pnpm-lock.yaml` file to control this.

Reply:
Thanks! Yes, we'll carefully consider the different lockfile formats and PNPM versions. Of course, we'll limit support to some reasonable range of lockfile formats and PNPM versions.

In any case, I'd recommend using the [packageManager](https://nodejs.org/api/packages.html#determining-package-manager) field in `package.json` for best support, since that gives Dependabot a hint of what version of PNPM will produce the most compatible result.